### ISSUE TITLE
Guidelines for values

### ISSUE THREAD

**Issue by roadscape** [bonustrackco/steemscript#1]
We will need further standards/guidelines for what each field should contain.

Some ideas to get started: 

 - maximum field lengths
   - *account.json_metadata.profile.name* (twitter: 20 chars)
   - *account.json_metadata.profile.about* (twitter: 160 chars) 
   - *account.json_metadata.profile.location* (twitter: 30 chars) 
   - *account.json_metadata.profile.website* (twitter: 100 chars) 

 - image dimensions/aspect ratios
   - *account.json_metadata.profile.profile_image*
     - profile images should be assumed to be square-cropped
     - optimal minimum size (230x230px?)
     - should be a valid URL, at minimum matching `^https?:\/\/`

**Comment by bonustrack** [comment]
I agree. I've added some small guideline here #3. Feel free to improve it.
